test/js/explore/explore-display-tabs.test.js

```diff
@@ -0,0 +1,281 @@
+// mock various modules from genome tab as these aren't being used, and throw compilation errors from jest
+jest.mock('components/explore/GenomeView', () => {
+  return {
+    igv: jest.fn(() => mockPromise)
+  }
+})
+
+jest.mock('components/visualization/RelatedGenesIdeogram', () => {
+  return {
+    Ideogram: jest.fn(() => mockPromise)
+  }
+})
+
+import { getEnabledTabs } from 'components/explore/ExploreDisplayTabs'
+
+// function to wrap test assertions
```

Comment:
You don't need to do property-by-property comparison -- you can just do
`expect(expectedResults).toEqual(getEnabledTabs(...))`
and that's probably even better because it will fail if there are stray unexpected properties

Reply:
Ah - you're right.  That will actually test the interface better.

Reply:
1926bcf